rero_ils/modules/items/api_views.py

```diff
@@ -49,6 +49,17 @@
 )
 
 
+def check_logged_user_authentication(func):
```

Comment:
Maybe better to replace this function to the related functions in `rero_ils/permissions.py`